_data/products.yml

```diff
@@ -1,8 +1,8 @@
-serverless: CockroachCloud Free (beta)
+serverless: CockroachCloud Serverless (beta)
```

Comment:
According to our final naming decisions, this should be `CockroachDB Serverless (beta)`.